Sources/SwiftFormat/PrettyPrint/PrettyPrint.swift

```diff
@@ -458,7 +458,12 @@ public class PrettyPrinter {
 
     // Print out the number of spaces according to the size, and adjust spaceRemaining.
     case .space(let size, _):
-      outputBuffer.enqueueSpaces(size)
+      if configuration.indentBlankLines, outputBuffer.isAtStartOfLine {
+        // An empty string write is needed to add line-leading indentation that matches the current indentation on a line that contains only whitespaces.
+        outputBuffer.write("")
```

Comment:
Don’t we also need to enqueue the spaces when `configuration.indentBlankLines` is set?

Reply:
Oh right, it seems we should enqueue this consistently, regardless of `configuration.indentBlankLines`.